homeassistant/components/datadog/issues.py

```diff
@@ -0,0 +1,42 @@
+"""Issues for Datadog integration."""
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
+
+from .const import DOMAIN
+
+
+async def deprecate_yaml_issue(
+    hass: HomeAssistant,
+    import_success: bool,
+) -> None:
+    """Create an issue to deprecate YAML config."""
+    if import_success:
+        async_create_issue(
+            hass,
+            DOMAIN,
```

Comment:
Use HOMEASSISTANT_DOMAIN instead so it uses the already existing strings there instead.